fe/fe-core/src/main/java/com/starrocks/catalog/JDBCResource.java

```diff
@@ -133,6 +133,9 @@ public String getProperty(String propertyKey) {
     protected void getProcNodeData(BaseProcResult result) {
         String lowerCaseType = type.name().toLowerCase();
         for (Map.Entry<String, String> entry : configs.entrySet()) {
+            if (entry.getKey().equalsIgnoreCase(PASSWORD)) {
```

Comment:
do we skip the config or output something but masked information, e.g. replace the real password with `******`

Reply:
I think skipping is simpler and better.